--- gnome-music-41.0/data/meson.build~	2021-09-19 12:11:14.000000000 +0200
+++ gnome-music-41.0/data/meson.build	2022-04-03 21:55:08.588671672 +0200
@@ -47,7 +47,6 @@
 appdata_conf.set('package_url', PACKAGE_URL)
 appdata_conf.set('package_url_bug', PACKAGE_URL_BUG)
 i18n.merge_file(
-  'appdata',
   input: configure_file(
     output: PROJECT_RDNN_NAME + '.appdata.xml.in',
     input: PROJECT_RDNN_NAME + '.appdata.xml.in.in',
